.clang-format

```diff
@@ -0,0 +1,17 @@
+﻿---
+BasedOnStyle: WebKit
+
+AlignAfterOpenBracket: AlwaysBreak
+AlignOperands: true
+AllowShortFunctionsOnASingleLine: Inline
+AlwaysBreakBeforeMultilineStrings: true
+AlwaysBreakTemplateDeclarations: Yes
+BinPackArguments: false
+BinPackParameters: false
+BreakBeforeBinaryOperators: None
+ColumnLimit: 100
+KeepEmptyLinesAtTheStartOfBlocks: false
+PenaltyBreakBeforeFirstCallParameter: 100
+PenaltyReturnTypeOnItsOwnLine: 1000
+SpaceInEmptyBlock: false
+...
```

Comment:
happy to see you are already using clang-format. you can think about integrating it with cmake such that just doing a `make clangformat` works. This is ofcourse optional.

Reply:
That is a great idea :) Implemented it